### ISSUE TITLE
How it works with  Batch Normalization?

### ISSUE THREAD

**Issue by kellenf** [SimonVandenhende/Multi-Task-Learning-PyTorch#28]
Using dataloader by dataloader, sometimes memory is limited, so only little batch for each dataloader. It will  disadvantage the performance of batch normalization? how to fix this problem?
